### PR TITLE
[TTAHUB-1151] add training vs technical assistance filter

### PR DESCRIPTION
**Pull request by thewatermethod**
## Description of change
Add filter for TTA type to the landing page and the regional dashboard
Note that this filter's query working 100% depends on the database cleanup being worked on in [this ticket](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1275).

The jira for the queue also necessitated a small change to the AR form language, which necessitated a small change to the e2e tests.

I took advantage of our typescript support and added the first new typescript file into our codebase - I ran into a few issues with the linter which I detailed a little more within the comments on this PR. You'll have to update your locally dependencies to see the benefit.

I also spent a bunch of time trying to figure out issues with the e2e tests (the issue that Jon eventually fixed) but this led me to do some cleanup on our tests, consolidating some of the files that contained one test into other suites. I think this is a little cleaner and might help fight the sprawl.

## How to test
Add a filter for "is" & "is not" each of the types. Spot check this info in the database. Confirm everything functions as desired.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1151

### Every PR

- [ ] Meets issue criteria
- [ ] JIRA ticket status updated
- [ ] Code is meaningfully tested
- [ ] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [ ] API Documentation updated
- [ ] Boundary diagram updated
- [ ] Logical Data Model updated
- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions
- [ ] UI review complete

### Production Deploy

- [ ] Staging smoke test completed

### After merge/deploy

- [ ] Update JIRA ticket status

### CODE REVIEW

File: frontend/src/pages/ActivityReport/Pages/activitySummary.js [hunk 1]
```diff
@@ -343,7 +343,7 @@ const ActivitySummary = ({
         <div id="tta" />
         <div className="margin-top-2">
           <FormItem
-            label="What TTA was provided"
+            label="What type of TTA was provided?"
             name="ttaType"
```
Comment (thewatermethod) [inline frontend/src/pages/ActivityReport/Pages/activitySummary.js:347]: This was an additional request by Adam Roux to synchronize filter and form question language

File: package.json [hunk 2]
```diff
@@ -121,15 +121,21 @@
   "eslintConfig": {
     "extends": [
       "airbnb-base",
```
Comment (thewatermethod) [inline package.json:123]: I ran into numerous eslint errors while writing the typescript for this and did a little research, it seems we needed to extend our base eslint config with this plugin. You'll need to re-up your dependencies locally.

File: src/lib/awsElasticSearch/index.js [hunk 3]
```diff
@@ -18,16 +18,16 @@ const generateEsConfig = () => {
       'aws-elasticsearch': [{
         credentials: {
           uri,
-          access_key,
-          secret_key,
+          access_key: accessKey,
+          secret_key: secretKey,
         },
```
Comment (thewatermethod) [inline src/lib/awsElasticSearch/index.js:23]: there are a few changes like this, which our new eslint config required

File: src/scopes/activityReport/index.test.js [hunk 4]
```diff
@@ -118,13 +116,6 @@ describe('filtersToScopes', () => {
     }, {
       silent: true,
     });
-    // Create ES client.
```
Comment (thewatermethod) [inline src/scopes/activityReport/index.test.js:121]: I moved this ES stuff into the test that requires it, as it was kind of noisy where it was when I was working on this test

**Review by kryswisnaskas — APPROVED** [review]
Looks good! 👍
 I verified the data filtered was correct for both the "is" and "is not" filters. It turns out that after the data cleanup, we will only have 2 ARs with null `ttaType `values.

Not related to this PR, but found during review was this:
https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1474

> I took advantage of our typescript support and added the first new typescript file into our codebase
🎉

**Review by GarrettEHill — APPROVED** [review]

**Pull request merged**